pkg/archive/compression/compression.go

```diff
@@ -135,8 +138,9 @@ const (
 )
 
 var (
-	gzipMagic = []byte{0x1F, 0x8B, 0x08}
-	zstdMagic = []byte{0x28, 0xb5, 0x2f, 0xfd}
+	gzipMagic  = []byte{0x1F, 0x8B, 0x08}
+	zstdMagic  = []byte{0x28, 0xb5, 0x2f, 0xfd}
+	bzip2Magic = []byte{0x42, 0x5A, 0x68}
```

Comment:
Who uses bzip2 for image archives? For what purpose?

Reply:
bzip2Magic is from moby ,I find moby support this format file , so I add it. [https://github.com/moby/moby/blob/06e3a49d66fa454e0124bd93570aac490bbf5544/pkg/archive/archive.go#L145](url)

Reply:
I'd prefer to avoid adding the bzip2 dependency (`compress/bzip2`) unless we have a specific use-case for it.

Reply:
Understood, we can introduce it again when needed next time, I will change it.